src/components/views/messages/SenderProfile.js

```diff
@@ -21,6 +21,18 @@ var React = require('react');
 module.exports = React.createClass({
     displayName: 'SenderProfile',
 
+    propTypes: {
+        mxEvent: React.PropTypes.object.isRequired, // event whose sender we're showing
+        aux: React.PropTypes.object, // stuff to go after the sender name, if anything
+        onClick: React.PropTypes.func,
+    },
+
+    getDefaultProps: function() {
+        return {
+            onClick: function() {},
```

Comment:
I don't think this is required, ftr. If it's omitted you'll just pass `undefined` to `span.onClick` which will do the right thing